benchmarks/synchrobench/src/main/java/com/oath/oak/synchrobench/maps/MyBufferOak.java

```diff
@@ -69,7 +69,7 @@ public int compareSerializedKeyAndKey(ByteBuffer serializedKey, MyBuffer key) {
             int cap1 = serializedKey.getInt(pos1);
             int pos2 = key.buffer.position();
             int cap2 = key.buffer.capacity();
-            return compare(serializedKey, pos1 + Integer.BYTES, cap1, key.buffer, pos2, cap2);
+            return compare(serializedKey, pos1 + Integer.BYTES, cap1 - Integer.BYTES, key.buffer, pos2, cap2);
```

Comment:
Also in line 47 (cannot review it because it didnt change) before return 0 check if cap2 > i.
I realize its not important for this benchmark because all keys are 100, but if someone ever looks at this code to learn and copy how to use oak it should be right.